vue-frontend/src/components/jobs-thankyou/ContributionSection.vue

```diff
@@ -0,0 +1,243 @@
+<template>
+  <div class="lg:tw-container tw-my-20 lg:tw-mb-40">
```

Comment:
Not looking good in 992 to 1400 devices, also container should be in > xl or 2xl devices only. Review other sections properly for all devices and implement like those only